src/icons/Status.js

```diff
@@ -183,12 +184,15 @@ export const StatusIconNoDefault = ({
         <Error className={className} />
     ) : loading ? (
         <Loading className={className} />
+    ) : info ? (
+        <Info className={className} />
     ) : null
 
 StatusIconNoDefault.propTypes = {
     valid: propTypes.bool,
     error: propTypes.bool,
     warning: propTypes.bool,
     loading: propTypes.bool,
+    info: propTypes.bool,
```

Comment:
I added the `info` prop to the `StatusIconNoDefault` because:
- It makes sense, since we also have an `Info` icon-component in `Status.js`
- It simplified the code in the `AlertBar` quite a bit